app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, class_name: 'Post', foreign_key: 'users_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'users_id'
+  has_many :comments, class_name: 'Comment', foreign_key: 'users_id'
+
```

Comment:
Please add `author_id` as a foreign key instead of `user_id` and can be seen in [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)